generic/tests/cfg/lvm.cfg

```diff
@@ -18,15 +18,15 @@
     clean = no
     check_mount = "mountpoint /mnt/kvm_test_lvm"
     fs_type = xfs
+    blk_extra_params_stg1 = "serial=TARGET_DISK1"
+    blk_extra_params_stg2 = "serial=TARGET_DISK2"
     RHEL.6:
         fs_type = ext4
     variants:
         - lvm_create:
             sub_type = lvm_create
             force_create_image_stg1 = yes
-            blk_extra_params_stg1 = "serial=TARGET_DISK1"
             force_create_image_stg2 = yes
-            blk_extra_params_stg2 = "serial=TARGET_DISK2"
             Host_RHEL.m6..ide:
```

Comment:
As we discussed, not affect test result on rhel6, so don't update it for rhel6.

Reply:
updated